data/lengths.csv

```diff
@@ -78,5 +78,6 @@ Unit,Value
 "the diameter of Saturn's rings", 250000000
 "the estimated sea level rise if the entire Antarctic ice sheet melted", 58.3
 "the supposed height of Disney Frozen's Olaf", 1.6256
-"the worlds longest conga line", 5.47
+"the worlds longest conga line", 25000
 "the height of the largest confirmed moose", 2.33
+"the giant space napkin", 384400000
```

Comment:
I feel this should be `the length of a giant space napkin, if a giant space napkin were, on average, to stretch from the surface of the earth to the surface of the moon`